hlxsites/merative2
# chore(logo): add curam logo

## Issue

Fixes N/A

## Description

**New**

- This PR adds the SVG of the new Curam logo
- This is in preparation for the Curam branding launch that we need to implement within Franklin for the **6/20** rollout.

## Design Specs

- Figma Link - https://www.figma.com/file/FSqQ1LYrVSssEFOZuOnMIE/Final-page-layouts-for-Merative.com?type=design&node-id=202%3A44111&t=NYMMsnfeQY0vyUCq-1

## Test URLs
  
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/drafts/putra/curam
- After (Changes from this PR): https://chore-add-curam-logo--merative2--proeung.hlx.page/drafts/putra/curam
  
## Testing Instruction

- Visit this draft/copy of the `Curam` tab of the masthead/nav - https://chore-add-curam-logo--merative2--proeung.hlx.page/drafts/putra/curam
- Add `:curam:` shortcode to the doc and ensure that the new Curam logo is displaying correctly.
